backend/src/main/java/com/cgi/retrospecto/backend/poker/controller/dto/CreateRoomDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.cgi.retrospecto.backend.poker.controller.dto;
+
+public class CreateRoomDTO {
```

Comment:
As mentioned in my createRoom comment I think this dto with only one field is not needed and it could be simpler if we just use the String.

Reply:
It is like this because the lack of my JS knowledge. So in frontend I could copy-paste working requests with some modifications. Later I will try to modify the frontend and simplify so these DTOs will be gone.